contracts/TreasuryActions.cdc

```diff
@@ -87,6 +87,8 @@ pub contract TreasuryActionsV2 {
                         .concat(amount.toString())
                         .concat(" ")
                         .concat(recipientVault.getType().identifier)
```

Comment:
We can remove the identifier of the vault

Reply:
What's the consideration behind it?